### ISSUE TITLE
Modify "Remove Styling" to warn of delete

### ISSUE THREAD

**Issue by qJake** [ankit/stylebot#337]
Please change the wording or add a confirmation prompt to the "Remove Styling" option for each page. I was developing some new CSS for a site and wanted to toggle my custom CSS on and off, and I thought this would temporarily remove/disable the stylesheet, **not delete it completely!**

It should either warn you with a prompt, or the option should say "Delete Styling". I think "Remove" is ambiguous (remove from the webpage, or remove from your PC?). I would also like to see an option to enable/disable all styles for a page on the extension menu, but fixing the wording is more important.

**Comment by futbalguy** [comment]
I just removed my styling after spending an hour on it because I also thought it was a temporary enable/disable toggle! Why is there even an option to "remove styling" in that menu? If it is going to delete the styling then it should be labelled as such and it probably should hidden deeper in the options menu!

**Comment by qJake** [comment]
Any word on this yet? I just re-installed Stylebot recently (there aren't very many good live CSS editors out there :confused:), and was disappointed to see that "Remove Styling" still permanently deletes any custom styles you have written.

**Comment by thebrodega** [comment]
Very disappointed that "Remove Styling" deletes the file. Why would you have a button that deletes all of your work right underneath the button that shares your work?  Opposites.

**Comment by ankit** [comment]
I will try and get to fixing this issue soon folks :(

**Comment by evitolins** [comment]
Any way to recover a deleted styling?  Clicked it and *poof* all my hard work gone.

**Comment by qJake** [comment]
Well, I hate to go this route... but if anyone is looking for a live CSS editor that doesn't delete your CSS so easily, check out [MagiCSS](https://chrome.google.com/webstore/detail/magicss-live-css-editor/ifhikkcafabcgolfjegfcgloomalapol?hl=en).

**Comment by feeei** [comment]
Hours of work disappeared because of a wording issue... yes, this thing that was first brought up in 2014 is still a problem in 2017.

It exists in two places: in the options menu and when you right-click the context menu on the upper right - which is where I encountered it. That button should have the world DELETE in it, should be RED and generally look dangerous to press. When I hovered over it, it temporarily deactivated the styling, which was a great feature. I thought pressing it would keep the stylesheet deactivated so I could move my mouse around while still having it deactivated. How would I know that it would DELETE the stylesheet changes when I pressed it? -.-

Will have nightmares about that "Remove Styling" button.
